app/models/post.rb

```diff
@@ -0,0 +1,2 @@
+class Post < ApplicationRecord
+end
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly implement relationship between `Post` and other models following the this [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)